src/main/java/org/kiwiproject/dropwizard/poller/health/ClientPollerHealthChecks.java

```diff
@@ -96,4 +104,75 @@ public static Result unhealthy(ClientPollerStatistics statistics, HealthStatus s
                 .withDetail(FAILURE_DETAILS_KEY, statistics.recentFailureDetails().collect(toUnmodifiableList()))
                 .build();
     }
+
+    /**
+     * Convenience method that registers the following health checks for the given poller and Dropwizard environment:
+     * {@link ClientPollerTimeBasedHealthCheck}, {@link ClientPollerLatencyBasedHealthCheck}, and
+     * {@link ClientPollerMissedPollHealthCheck}
+     *
+     * @param poller      The poller being monitored
+     * @param environment The dropwizard environment
+     */
+    public static void registerPollerHealthChecks(ClientPoller poller, Environment environment) {
+        registerPollerHealthChecks(poller, environment, PollerHealthCheckConfig.builder().build());
+    }
+
+    /**
+     * Convenience method that registers the following health checks for the given poller and Dropwizard environment:
+     * {@link ClientPollerTimeBasedHealthCheck}, {@link ClientPollerLatencyBasedHealthCheck}, and
+     * {@link ClientPollerMissedPollHealthCheck}
+     *
+     * @param poller            The poller being monitored
+     * @param environment       The dropwizard environment
+     * @param healthCheckConfig The config for the health checks
+     */
+    public static void registerPollerHealthChecks(ClientPoller poller,
+                                                  Environment environment,
+                                                  PollerHealthCheckConfig healthCheckConfig) {
+
+        var pollerHealthChecks = buildPollerHealthChecks(poller, healthCheckConfig);
+        pollerHealthChecks.forEach(healthCheck -> registerPollerHealthCheck(healthCheck, environment));
+    }
+
+    private static void registerPollerHealthCheck(PollerHealthCheck pollerHealthCheck, Environment environment) {
+        environment.healthChecks().register(pollerHealthCheck.getName(), pollerHealthCheck.getHealthCheck());
+    }
+
+    /**
+     * Build the standard client poller health checks, returning a list containing the actual health checks wrapped in a
+     * {@link PollerHealthCheck}. Uses a default {@link PollerHealthCheckConfig}.
+     *
+     * @param poller The poller being monitored
+     * @return a list of health checks for the poller
+     */
+    public static List<PollerHealthCheck> buildPollerHealthChecks(ClientPoller poller) {
+        return buildPollerHealthChecks(poller, PollerHealthCheckConfig.builder().build());
+    }
+
+    /**
+     * Build the standard client poller health checks, returning a list containing the actual health checks wrapped in a
```

Comment:
Should we state explicitly that the returned list is _mutable_? Or, should we actually be returning an unmodifiable list, e.g. `List.of`?

Reply:
I made it List.of